frontend/packages/dev-console/src/utils/pipeline-plugin-utils.ts

```diff
@@ -28,7 +28,7 @@ export const tknPipelineAndPipelineRunsResources = (namespace: string): Firehose
 };
 
 type PipelineItem = {
-  pipeline: K8sResourceKind[];
+  pipelines: K8sResourceKind[];
```

Comment:
@sahil143 I renamed this variable because it is a list and thus should have an `s` on the end. Sorry that I missed this when doing your review.